sample/repeat-fetch.c

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2019-2020 NetFoundry, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * \file repeat-fetch.c
+ * \brief demonstrates re-connecting usage of HTTP client
+ */
+
+#include <uv_mbed/um_http.h>
+#include "common.h"
+
+int count = 5;
+um_http_t time_clt;
+uv_timer_t time_timer;
+
+static void timer_cb(uv_timer_t *timer) {
+    if (count-- > 0) {
+        printf(">>> calling time service count left = %d\n\n", count);
+        um_http_req_t *req = um_http_req(&time_clt, "GET", "/api/timezone/EST");
```

Comment:
would it be nice to declare or find/reuse constants for stuff like "GET", "POST" etc?